.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
i tried to reverse engineer your prettier settings but probably didn't get it 100% right. happy to use whatever you want -- just let me know so the project stays consistent.